src/constants.js

```diff
@@ -1,12 +1,21 @@
 module.exports = {
-  CONTROL_BUTTON_CLASS: 'mapbox-gl-draw_ctrl-draw-btn',
-  CONTROL_BUTTON_LINE_CLASS: 'mapbox-gl-draw_line',
-  CONTROL_BUTTON_POLYGON_CLASS: 'mapbox-gl-draw_polygon',
-  CONTROL_BUTTON_POINT_CLASS: 'mapbox-gl-draw_point',
-  CONTROL_BUTTON_TRASH_CLASS: 'mapbox-gl-draw_trash',
-  CONTROL_GROUP_CLASS: 'mapboxgl-ctrl-group',
-  ATTRIBUTION_CLASS: 'mapboxgl-ctrl-attrib',
-  ACTIVE_BUTTON_CLASS: 'active',
+  classes: {
+    CONTROL_BASE: 'mapboxgl-ctrl',
+    CONTROL_PREFIX: 'mapboxgl-ctrl-',
```

Comment:
oh, wow. Maybe a good difference between constants and string literals is that they can be easier to distinguish.